app/controllers/spotlight_search/export_jobs_controller.rb

```diff
@@ -0,0 +1,12 @@
+module SpotlightSearch
+  class ExportJobsController < ApplicationController
+    include SpotlightSearch::Concerns::Exportable
+
+    def export_job
```

Comment:
We need to get the model_name, All the filters applied through JS and convert it and create records.

Reply:
I was thinking since we have the filtered records, we'll just records.class to get the model name